integrations/jenkins/main.py

```diff
@@ -49,6 +53,9 @@ async def handle_events(event: dict[str, Any]) -> dict[str, bool]:
 
     if kind:
         resource = await jenkins_client.get_single_resource(event["url"])
+        if kind == ObjectKind.BUILD:
+            # the only way to get the job name is from the build url
+            resource = {**resource, "__jobName": parse_job_name(resource["url"])}
```

Comment:
We still have to parse the URL in order to get the job name for a build  webhook event.

Reply:
Would you rather we use the `fullDisplayName`. I am more wary of its change. I am more certain of the `/job/<jobName>/` since the <jobName> is from the "name" field of the job json data